.changeset/lcm-task-bridge-suggestions.md

```
@@ -0,0 +1,5 @@
+---
+"@martian-engineering/lossless-claw": patch
+---
+
+Add inert experimental task-bridge suggestion storage for future observed-work review flows.
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)